src/main/java/com/shaft/api/RestActions.java

```diff
@@ -649,14 +639,14 @@ private static boolean compareJSONEquals(org.json.simple.JSONObject expectedJson
 
     @SuppressWarnings("unchecked")
     private static boolean compareJSONContains(Response response, org.json.simple.JSONObject expectedJsonObject,
-                                               org.json.simple.JSONArray expectedJsonArray, org.json.simple.JSONObject actualJsonObject,
-                                               org.json.simple.JSONArray actualJsonArray, String jsonPathToTargetArray)
+                                               JSONArray expectedJsonArray, org.json.simple.JSONObject actualJsonObject,
+                                               String jsonPathToTargetArray)
             throws JSONException, ParseException {
         JSONParser parser = new JSONParser();
         if (!jsonPathToTargetArray.equals("") && (expectedJsonArray != null)) {
             // if expected is an array and the user provided the path to extract it from the
             // response
-            actualJsonArray = (org.json.simple.JSONArray) parser
+            org.json.simple.JSONArray actualJsonArray = (org.json.simple.JSONArray) parser
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'org.json.simple.JSONArray' due to existing import 'org.json.simple.JSONArray'](https://app.codacy.com/manual/mohab.mohieeldeen/SHAFT_ENGINE/pullRequest?prid=5737302)